tutorials/dataframe/df104_HiggsToTwoPhotons.py

```diff
@@ -175,7 +177,7 @@
 ratiobkg.GetYaxis().SetNdivisions(503, False)
 ratiobkg.GetYaxis().ChangeLabel(-1, -1, 0)
 ratiobkg.GetXaxis().SetTitle("m_{#gamma#gamma} [GeV]")
-ratiobkg.Draw()
+ratiobkg.Draw("AXIS")
```

Comment:
Usually the AXIS option is used internally only ...

Reply:
But fine ..

Reply:
@linev following Olivier's comment, why do we need "AXIS" here?

Reply:
"AXIS" draw only histogram axes without content. Also no stats box will be created.
And it is [fully legitime](https://root.cern/doc/master/classTHistPainter.html#HP01a) draw option

Reply:
Ok then I'm confused by @couet 's statement :smile:

Reply:
`TF1` internally uses exactly this method to draw axes. Problem of solution with `TF1` - not all properties will be correctly preserved when `TCanvas` stored in the ROOT file. Therefore I would always recommend to use `TH1` class for the case when just axis should be drawn